cms/envs/common.py

```diff
@@ -819,6 +819,9 @@
     'openedx.core.djangoapps.coursetalk',  # not used in cms (yet), but tests run
     'xblock_config',
 
+    # Maintenance tools
```

Comment:
@adampalay @awaisdar001 I also added maintenance app to INSTALLED_APPS which I missed earlier.